packages/driver/cypress/fixtures/issue-29093.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en" class="dark">
+
+<head>
+  <meta charset="utf-8" />
+  <title>29093 repro</title>
+  <style id="compiled-css" type="text/css">
+    .big {
+      font-size: 128px;
+    }
+
+    .contents {
+      display: contents
```

Comment:
This is what is causing Cypress to not properly determine that `#sidebar-left` is not hidden in the fixture. `display: contents` causes the element to not generate its own box, which prevents overflow:hidden from being applied as expected - the element is effectively replaced by its children, and its box model (and overflow settings) should be disregarded. See: https://drafts.csswg.org/css-display/#valdef-display-contents.

Reply:
It's true. Check moved to canClip.